File: web/tests/reference/intervals/minimum-arrows-to-burst-balloons.py
def find_min_arrow_shots(points):
    points.sort(key=lambda b: b[1])
    arrows = 1
    arrow_x = points[0][1]
    for start, end in points[1:]:
        if start > arrow_x:
            arrows += 1
            arrow_x = end
    return arrows

File: web/tests/reference/intervals/minimum-arrows-to-burst-balloons.js
function findMinArrowShots(points) {
  points.sort((a, b) => a[1] - b[1]);
  let arrows = 1;
  let arrowX = points[0][1];
  for (const [start, end] of points.slice(1)) {
    if (start > arrowX) {
      arrows++;
      arrowX = end;
    }
  }
  return arrows;
}
